app/models/user.rb

```diff
@@ -13,6 +13,7 @@ class User < ApplicationRecord
   has_many :user_roles, dependent: :destroy
   has_many :forms, through: :user_roles, primary_key: "form_id"
   has_many :collections, through: :organization
+  has_many :personas
```

Comment:
the User won't be attached to a Persona. so, this can be removed.

More-formally assigning/identifying persona-type roles is an interesting idea. (Could be great for formalized-workflows)
But, we don't have a requirement to set, or for users to set their own Persona.

The Persona objects will be co-managed in the CX process for the first round of service definitions (happening this year). Agency Service Managers will be able to associate stages with Personas.

Reply:
I was curious about you set user_id as required for persona.   Maybe we can discuss at tomorrows standup how these persona's are going to be used?   If user_id is required for a persona then it seems like the has_many/belongs_to between users and persona should be explicit.

Reply:
oh, the `user_id` was for indicating the user who created the Persona. this will likely change in a similar way to Websites, where multiple people can manage one.

the "ownership" or "governance" around the Personas are still evolving.

Reply:
Ok,  I removed the relationship between persona and user.